src/spikeinterface/core/recording_tools.py

```diff
@@ -917,3 +917,34 @@ def get_rec_attributes(recording):
         dtype=recording.get_dtype(),
     )
     return rec_attributes
+
+
+def check_recording_attributes_match(recording1, recording2_attributes) -> bool:
```

Comment:
Sorry based on our previous discussions should we name this

do_recording_attributes_match

so it is a question and the bool is the answer?

Reply:
done